DESIGN.md

```diff
@@ -24,7 +24,7 @@ It also includes recommendations for future implementers.
 
 We set for the following goals (mostly taken from
 [brittany](https://github.com/lspitzner/brittany)):
-* Preserve the meaning of the formatted functions;
+* Preserve the meaning of the formatted functions when no CPP is used;
```

Comment:
There is a discrepancy in the list that appear here and in the readme, maybe they should be unified somehow.